wrappers/nodejs/scripts/generate-dist-package.js

```diff
@@ -0,0 +1,50 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+//   Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+
+// Purpose: pack librealsense dir & wrappers/nodejs dir into npm package
+
+const {exec} = require('child_process');
+const package = require('../package.json');
+const jf = require('jsonfile');
```

Comment:
Can we use same pattern? w/ and w/o`{}`

Reply:
{exec} means to make new var `exec` equals `child_process.exec` :)

Yep I know, yet another new syntax in new JavaScript...

Reply:
THanks for the explanation, so use same syntax or different? I know I'm a little picky.

Reply:
Different, unless writing `const exec = require('child_process').exec;`

Reply:
This is kinda like the ancient `with` syntax in JS